Documentation/concepts/kubernetes/compatibility.rst

```diff
@@ -23,4 +21,101 @@ All Kubernetes versions listed are compatible with Cilium:
++-----------------+----------------+
+| 1.6.0-rc1       | 1.14           |
++-----------------+----------------+
+| 1.6.0-rc2       | 1.14           |
++-----------------+----------------+
+| 1.6.0-rc3       | 1.14           |
++-----------------+----------------+
+| 1.6.0-rc4       | 1.14           |
++-----------------+----------------+
+| 1.6.0-rc5       | 1.14           |
++-----------------+----------------+
+| 1.6.0-rc6       | 1.14           |
++-----------------+----------------+
+| 1.6.0-rc7       | 1.14           |
++-----------------+----------------+
+| 1.6.0           | 1.14           |
++-----------------+----------------+
+| 1.6.1           | 1.14           |
++-----------------+----------------+
+| 1.6.2           | 1.14           |
++-----------------+----------------+
+| 1.6.3           | 1.14           |
++-----------------+----------------+
+| 1.6.4           | 1.14           |
++-----------------+----------------+
+| 1.6.5           | 1.14           |
++-----------------+----------------+
+| 1.6.6           | 1.14           |
++-----------------+----------------+
+| 1.6.7           | 1.14           |
++-----------------+----------------+
+| 1.6.8           | 1.14           |
++-----------------+----------------+
+| 1.6.9           | 1.15           |
++-----------------+----------------+
+| 1.6.10          | 1.15           |
++-----------------+----------------+
+| 1.6             | 1.15.1         |
++-----------------+----------------+
+| 1.7.0-rc1       | 1.15           |
++-----------------+----------------+
+| 1.7.0-rc2       | 1.16           |
++-----------------+----------------+
+| 1.7.0-rc3       | 1.16           |
++-----------------+----------------+
+| 1.7.0-rc4       | 1.16           |
++-----------------+----------------+
+| 1.7.0           | 1.16           |
++-----------------+----------------+
+| 1.7.1           | 1.16           |
++-----------------+----------------+
+| 1.7.2           | 1.16           |
++-----------------+----------------+
+| 1.7.3           | 1.17           |
++-----------------+----------------+
+| 1.7.4           | 1.17           |
++-----------------+----------------+
+| 1.7.5           | 1.17           |
++-----------------+----------------+
+| 1.7.6           | 1.18           |
++-----------------+----------------+
+| 1.7             | 1.18.1         |
++-----------------+----------------+
+| 1.8.0-rc1       | 1.19           |
++-----------------+----------------+
+| 1.8.0-rc2       | 1.20           |
++-----------------+----------------+
+| 1.8.0-rc3       | 1.20           |
++-----------------+----------------+
+| 1.8.0-rc4       | 1.20           |
++-----------------+----------------+
+| 1.8.0           | 1.21           |
++-----------------+----------------+
+| 1.8.1           | 1.21           |
++-----------------+----------------+
+| 1.8             | 1.21.1         |
++-----------------+----------------+
+| latest / master | 1.22.1         |
```

Comment:
Should this be updated every time we create new CRD version?

Reply:
That's a good point... I think yes, it makes sense.